[AOTInductor] Prepare for ProxyExecutor, OSS only change

SherlockNoMad opened the pull request:
Summary: Minor fixes to export schema and serialization

Test Plan: OSS CI

Differential Revision: D48280809

zhxchen17 commented on torch/_export/serde/schema.py:
```diff
@@ -152,6 +159,9 @@ class Argument(_Union):
     as_graph: GraphArgument
     as_optional_tensors: List[OptionalTensorArgument]
 
+    def __repr__(self):
+        return super().__repr__()
+
```
Why we need this? won't Python look up automatically for __repr__?

SherlockNoMad replied:
dataclass generates a repr function, which overrides parent class' implementation. 
I guess I can also fix this by setting (repr=False)...

SherlockNoMad committed:
[AOTInductor] Prepare for ProxyExecutor, OSS only change (#107065)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/107065

Minor fixes to export schema and serialization

Test Plan: OSS CI

Reviewed By: zhxchen17

Differential Revision: D48280809

fbshipit-source-id: 40c0110c6743c61aca7ad837a48b6e14b2917800

SherlockNoMad commented:
@pytorchbot merge
